godel_surface_detection/include/services/surface_blending_service.h

```diff
@@ -124,8 +136,15 @@ class SurfaceBlendingService
   bool select_surface_server_callback(godel_msgs::SelectSurface::Request& req,
                                       godel_msgs::SelectSurface::Response& res);
 
-  bool process_path_server_callback(godel_msgs::ProcessPlanning::Request& req,
-                                    godel_msgs::ProcessPlanning::Response& res);
+
+  void process_planning_action_callback(const godel_msgs::ProcessPlanningGoalConstPtr &goal);
```

Comment:
I don't mean to be anal, but all the functions you added except this one follow the ROS naming standard.